admin-dev/themes/new-theme/js/pages/catalog/product/specific-price-form-handler.js

```diff
@@ -154,6 +154,7 @@ class SpecificPriceFormHandler {
     var selectorPrefix = this.getPrefixSelector(usePrefixForCreate);
 
     $('#form_modal_cancel').click(() => this.closeEditPriceModalAndRemoveForm());
+    $('#form_modal_close').click(() => this.closeEditPriceModalAndRemoveForm());
```

Comment:
You also can do this, but not necessary.
`$('#form_modal_cancel, #form_modal_close').on('click', (() =>...`